example/src/pages/Pictograms.tsx

```diff
@@ -59,16 +58,14 @@ const sortPictogramSet = (
 
 const filteredIOPictograms = filterPictogramSet(
   {
-    ...IOPictogramsObject,
-    ...IOPictogramsLegacy
+    ...IOPictogramsObject
```

Comment:
What do you think to pass the object without spreading it
```suggestion
const filteredIOPictograms = filterPictogramSet(
  IOPictogramsObject,
  IOPictograms
);
```

Reply:
Good catch! It made sense with more than two objects to filter. But we can just put the single object when it's just one. Fixed in the following commit → [`d44a226` (#384)](https://github.com/pagopa/io-app-design-system/pull/384/commits/d44a226853bf6ca0841f0aeab9078cb6a22918cf)